packages/components/Menu/src/MenuGroup/MenuGroup.styling.ts

```diff
@@ -0,0 +1,19 @@
+import type { UseStylingOptions } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+
+import { menuGroupName } from './MenuGroup.types';
+import type { MenuGroupProps, MenuGroupTokens, MenuGroupSlotProps } from './MenuGroup.types';
+
+export const stylingSettings: UseStylingOptions<MenuGroupProps, MenuGroupSlotProps, MenuGroupTokens> = {
+  tokens: [menuGroupName],
+  slotProps: {
+    root: buildProps(
```

Comment:
nit: You can just pass an object, you don't need to call build props.

Reply:
Yep confirmed that display is flex by default - https://reactnative.dev/docs/layout-props#display. I ended up removing the styling file altogether as it seems unnecessary